### PR TITLE
Add new alert for samples rejected due ti duplicate timestamp

### PR DESCRIPTION
**Pull request by sebglon**
Add new alert for prometheus.

show issue: #1139

### CODE REVIEW

**Comment by sebglon** [comment]
@gianrubio Yes sorry. and bump helm version.

**Comment by gianrubio** [comment]
Ci is failling, could you fix that?

**Comment by sebglon** [comment]
I don't understand what is the error?

**Comment by sebglon** [comment]
That's goog now

File: contrib/kube-prometheus/assets/prometheus/rules/prometheus.rules.yaml [hunk 1]
```diff
@@ -90,3 +90,12 @@ groups:
     annotations:
       description: "Prometheus {{ $labels.namespace }}/{{ $labels.pod}} isn't ingesting samples."
       summary: "Prometheus isn't ingesting samples"
+
+  - alert: PrometheusTargetScapesDuplicate
+    expr: increase(prometheus_target_scrapes_sample_duplicate_timestamp_total[2h]) > 0
+    for: 4h
```
Comment (brancz) [inline contrib/kube-prometheus/assets/prometheus/rules/prometheus.rules.yaml]: This is a pretty long time range, `10m` would over a `[5m]` range query would give the same result but you'd be alerting much quicker.

Comment (sebglon) [reply contrib/kube-prometheus/assets/prometheus/rules/prometheus.rules.yaml]: Yep nice

**Comment by sebglon** [comment]
That's seems good

**Comment by brancz** [comment]
Leaving final sign-off to @gianrubio for the helm changes.

**Comment by gianrubio** [comment]
Thanks @sebglon

**Pull request merged**